src/Medulla/Kubernetes/Operator/Startup.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the Medulla Contributors under one or more agreements.
+// The Medulla Contributors licenses this file to you under the Apache 2.0 license.
+// See the LICENSE file in the project root for more information.
+
+using KubeOps.Operator;
+
+namespace Controller;
+
+public class Startup
+{
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddKubernetesOperator();
+    }
+
+    public void Configure(IApplicationBuilder app)
+    {
+        app.UseKubernetesOperator();
+    }
+}
```

Comment:
This is the old method of bootstrapping an ASP.NET app, lets move all this logic to [top level statements](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/top-level-statements) inside of Program.cs

You can see how this is done in other Program.cs files in the repo.

By doing this this entire file should end up deleted.

Reply:
@AndrewBabbitt97  Just pushed up the latest changes regarding this by following this documentation here: https://docs.microsoft.com/en-us/aspnet/core/migration/50-to-60?view=aspnetcore-6.0&tabs=visual-studio. Let me know if I missed something.

Reply:
Marking this as resolved unless further updates are needed.